cp3pt0-deployment/common/utils.sh

```diff
@@ -333,7 +333,7 @@ function wait_for_nss_patch() {
     fi
     
     # wait for deployment to be ready
-    local deployment_name=$(${OC} get deployment -n ${namespace} -l operators.coreos.com/${package_name}.${namespace}='' --no-headers | awk '{print $1}')
+    local deployment_name="ibm-common-service-operator"
```

Comment:
Why are we changing this, as it does not seem related to Reporter?

Reply:
it was merge from origin repo